EndlessSky.xcodeproj/project.pbxproj

```diff
@@ -907,6 +905,45 @@
 				8E8A4C648B242742B22A34FA /* Weather.cpp */,
 				F8C14CFB89472482F77C051D /* Weather.h */,
 				0C90483BB01ECD0E3E8DDA44 /* WeightedList.h */,
+				950742538F8CECF5D4168FBC /* EsUuid.cpp */,
+				86AB4B6E9C4C0490AE7F029B /* EsUuid.h */,
+				499B4DA7A9C7351120660643 /* MaskManager.cpp */,
+				191E4107A4F1CBBC2526A0E9 /* MaskManager.h */,
+				A00D4207B8915B5E7E9B4619 /* RandomEvent.h */,
+				9F0F4096A9008E2D8F3304BB /* GameAction.cpp */,
+				6833448DAEB9861D28445DD5 /* GameAction.h */,
+				389245138CF297EB417DF730 /* UniverseObjects.cpp */,
+				DB9A43BA91B3BC47186BF05E /* UniverseObjects.h */,
+				A2A948EE929342C8F84C65D1 /* TestContext.h */,
+				A3134EC4B1CCA5546A10C3EF /* TestContext.cpp */,
+				A7E640C7A366679B27CCADAC /* GameLoadingPanel.cpp */,
+				DC8146D5A145C2DA87D98F1F /* GameLoadingPanel.h */,
+				6A4E42FEB3B265A91D5BD2FC /* TextReplacements.cpp */,
+				86D9414E818561143BD298BC /* TextReplacements.h */,
+				74F543598EEFB3745287E663 /* Bitset.cpp */,
+				210C41C0BA33F71A694D5F98 /* Bitset.h */,
+				A3754152958FBC924E9F76A9 /* opengl.cpp */,
+				EB634E95A88454ADDB8644B1 /* opengl.h */,
+				E8F645ACA30BA0E95F83803C /* FireCommand.cpp */,
+				EB4645F28765D37290EA6F78 /* FireCommand.h */,
+				46B444A6B2A67F93BB272686 /* ByGivenOrder.h */,
+				DE844E2BBF82C39B568527CB /* ByName.h */,
+				C62B4D15899E5F47443D0ED6 /* DamageProfile.cpp */,
+				19184B47B496B414EC9CE671 /* DamageProfile.h */,
+				5CF247B48EEC7A3C366C1DFA /* DamageDealt.h */,
+				D0FA4800BE72C1B5A7D567B9 /* MenuAnimationPanel.cpp */,
+				8CB543A1AA20F764DD7FC15A /* MenuAnimationPanel.h */,
+				32404878BFF00A095AF2DD2C /* MapPlanetCard.cpp */,
+				A2A94C40A8DCA99809AD5DEE /* MapPlanetCard.h */,
+				61FA4C2BB89E08C5E2B8B4B9 /* Variant.cpp */,
+				5EBC44769E84CF0C953D08B3 /* Variant.h */,
+				086E48E490C9BAD6660C7274 /* ExclusiveItem.h */,
+				7CE64A4BA79C2C654848F558 /* TaskQueue.cpp */,
+				47304B4DBCD8D319692647D2 /* TaskQueue.h */,
+				341645FAA25F7901221B8965 /* PrintData.cpp */,
+				CCE547DBB6C74E18E1B84D29 /* PrintData.h */,
+				A7F240B4AC219841424A387A /* Logger.cpp */,
+				AE57401AA43232EDEABFAE13 /* Logger.h */,
```

Comment:
This list contains some duplicate entries (probably due to interaction with another PR that got merged that sorted this list alphabetically).